openedx/core/djangoapps/schedules/templates/schedules/edx_ace/recurringnudge_week1/email/body.html

```diff
@@ -0,0 +1,3 @@
+Dear {{student_name}},
```

Comment:
So that theming overrides can set the template `schedules/edx_ace/recurringnudge_week1/email/body.html` to override this template. Django doesn't give app templates separate namespaces by default, afaik.